tests/unit/cisco/aireos/test_cisco_aireos.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+import pytest
+
+from scrapli_community.cisco.aireos.cisco_aireos import DEFAULT_PRIVILEGE_LEVELS
+
+
+# TODO
```

Comment:
I think you've done it 😁

Reply:
Sorry, that's me being a diligent little programmer so I don't forget 🤠